account-integrations/safe/src/SafeZkEmailRecoveryPlugin.sol

```diff
@@ -79,7 +111,7 @@ contract SafeZkEmailRecoveryPlugin {
      * @param dkimPublicKeyHash hash of DKIM public key - keccak256(abi.encodePacked(dkimPublicKey))
      * @param dkimRegistry address of a user-defined DKIM registry
      */
-    function addRecoveryHash(
+    function configureRecovery(
```

Comment:
Good to have an optional field here to set a custom delay as part of the initial recovery config. Saves a txn/UserOp if someone wants a custom delay.

I would still keep `setCustomDelay` in case someone wants to update it, such as in your example if they go on vacation or are away for some reason.

Reply:
Added